public/js/Histogram.js

```diff
@@ -0,0 +1,100 @@
+var d3 = require('d3');
+var _ = require('lodash');
+_.mixin(require('lodash-deep'));
+
+function Histogram(sampleData, element, featureNames) {
```

Comment:
Also, don't sanitise the story and make it a sterile technical manual about classes and objects in JS. Start with something like, "My MSc advisor, @jni, is totally clueless when it comes to JS, so he asked me this noob question the other day..." Ok maybe not exactly that but you get my drift. I hate that scientific writing tries to erase the humans from the equation. It's supposed to make it more "objective", but clearly it is worthless since something like 60-90% of scientific studies don't reproduce... So imho you might as well bring back the human element.

`</rant>`